Don't include timestamp in generated files since this prevents muniversal
portgroup from being able to merge them correctly.
--- tool/jhton.c.orig	2005-02-15 15:06:07.000000000 -0600
+++ tool/jhton.c	2014-10-19 14:38:06.000000000 -0500
@@ -674,7 +674,6 @@
 	char *	pagesection;		// such as "3X".
 	char	lcletter;		// manual tab section, such as "j".
 	char *	revision;		// from centered table datum.
-	time_t	currtime;		// for ctime().
 
 // Extract "weird" header values:
 //
@@ -690,11 +689,10 @@
 		  "contain revision information");
 	}
 
-// Emit file header; note, ctime() output already contains a newline:
+// Emit file header:
 
-	(void) time(&currtime);
-	(void) printf(".\\\" Auto-translated to nroff -man from %s by %s at %s",
-		      Filename, gc_myname, ctime(&currtime));
+	(void) printf(".\\\" Auto-translated to nroff -man from %s by %s\n",
+		      Filename, gc_myname);
 
 	(void) printf(".\\\" %s\n",  filerev);
 	(void) printf(".TA %c\n",    lcletter);
